src/ray/object_manager/object_manager.cc

```diff
@@ -384,19 +386,19 @@ void ObjectManager::ProcessNewClient(TcpClientConnection &conn) {
 void ObjectManager::ProcessClientMessage(std::shared_ptr<TcpClientConnection> &conn,
                                          int64_t message_type, const uint8_t *message) {
   switch (message_type) {
-  case object_manager_protocol::MessageType_PushRequest: {
+  case static_cast<int64_t>(object_manager_protocol::MessageType::PushRequest): {
```

Comment:
Would it make more sense to do `static_cast<object_manager_protocol::MessageType>(message_type)` in the `switch` statement instead of in all of the `case` statements?

Reply:
Same comment/question about a number of other places.

Reply:
Hi Robert, for the 2 places where contains switch statements, they both have a case condition `protocol::MessageType::DisconnectClient` which does not belong to most of other cases from `object_manager_protocol::MessageType`. If I use `static_cast` in `switch`, then I need to use `static_cast<MessageType>(CommonMessageType::DISCONNECT_CLIENT)` which will cause error because there is no 0 in `MessageType`. Therefore, there are 2 options to solve this problem. The first one is the way I used in my code change. The other way is to add an enum element which equals to 0 in local_scheduler.fbs and object_manager.fbs. Do you think the second way is better?

Reply:
Oh, that makes sense. In that case, I think the way you did it makes sense.